php/mixphp-workerman/bin/workerman.php

```diff
@@ -0,0 +1,8 @@
+<?php
+require __DIR__ . '/../vendor/autoload.php';
+
+$vega = App\Vega::new();
+$http = new Workerman\Worker("http://0.0.0.0:3000");
+$http->onMessage = $vega->handler();
+$http->count = (int)(shell_exec('nproc') ?: 8) * 2;
```

Comment:
seems not consistent with `swoole_cpu_num() * 2` configured for **swoole**

Reply:
He is mixphp-workerman here, use `workerman`, not `swoole`